.env.sample

```diff
@@ -23,22 +23,27 @@ TAIKO_TOKEN_L1_ADDRESS=0x10dea67478c5F8C5E2D90e5E9B26dBe60c54d800
 PRECONFIRMATION_WHITELIST=0xFD019460881e6EeC632258222393d5821029b2ac
 TAIKO_ANCHOR_ADDRESS=0x1670000000000000000000000000000000010001
 
-# Node Verbosity, set to 4 to debug peers for preconfirmations.
+# Node Verbosity
 VERBOSITY=3
 
 # P2P
 DISABLE_P2P_SYNC=false
 P2P_SYNC_URL=https://rpc.mainnet.taiko.xyz
 
 ## Driver configs for preconfirmations
-# IP address where your node can be reached from at port 4001
-# which will be used to join the P2P network and received preconfirmation blocks
+# Enable preconfs p2p network, defaults to true. If set to false, your node will NOT join the P2P network and receive preconf blocks.
+ENABLE_PRECONFS_P2P=true
+# IP address where your node can be reached which will be used to join the P2P network and received preconfirmation blocks
 PUBLIC_IP=
 # TCP Port used for discovery on the p2p network, it MUST be open and external and accessible via the same IP address as the public IP.
 P2P_TCP_PORT=4001
 # UDP Port used for discovery on the p2p network, it MUST be open and external and accessible via the same IP address as the public IP.
 P2P_UDP_PORT=30303
 # You may have to configure port forwarding rules in your firewall/router: if so, simply add rules for the above ports.
+# Private Keys for Peer ID: One must be set to persist your peer ID.
+# This private key does not need to hold eth or any valuables, it is simply to persist an identity
```

Comment:
add a comment saying the path to the private key has to be an actual path to a .txt file that consists entirely of a private key with no newlines, PRIV_RAW is just the hexstring private key in plaintest